### DISCUSSION TITLE
Open Source Commitment

### DISCUSSION THREAD

**Discussion by AskBojesen** [dotnet-bluetooth-le/dotnet-bluetooth-le#909] [General]
Hi my friends! I know that I have been away/passive in this great plugin.

A think the idea and concept of a project like this is contributing to making me believe in a future for this world.

This plugin saved  our Android/iPhone Maui app our small private startup company Egoo.Health

I have been in the private IT sector in Denmark for 20+ years working with desktop applications for Windows - so I know my Windows, and I have seldom had a satisfaction this like making BLE work for my device.

**BUT!**

I simply don't have time to contribute more right now due to business in personal life and work.

So @janusw , don't count me in for review etc. until I get more time...

Keep up the good work!

**Comment by janusw** [comment]
Hi Ask,

I'm sorry to hear that, but of course I understand. Thanks for your clear communication here, and of course for all your contributions to the project (which were quite significant)!
Making a clear cut is certainly better than just silently dropping out (which happens way too often with open-source projects). I'm often struggling myself to find some time, but so far it still works out every once in a while.

So, I wish you all the best for the future, and whenever you feel the urge to get back into the project, you're more than welcome to contribute! 😊
